src/Compilers/Core/Portable/Emit/EditAndContinue/DeltaMetadataWriter.cs

```diff
@@ -37,6 +37,18 @@ internal sealed class DeltaMetadataWriter : MetadataWriter
         private readonly EventOrPropertyMapIndex _propertyMap;
         private readonly MethodImplIndex _methodImpls;
 
+        // For the EncLog table we need to know which things we're emitting custom attributes for so we can
+        // correctly map the attributes to row numbers of existing attributes for that target
+        private readonly List<EntityHandle> _customAttributeTargets = new List<EntityHandle>();
```

Comment:
> = new List<EntityHandle>() [](http://example.com/codeflow?start=67&length=27)

Minor: For consistency with the other fields, consider assigning in the constructor instead.

Reply:
Thanks, I should have checked after moving it